<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gestor de Suscripciones</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Estilo general del fondo */
QWidget#Login {
    background-color: #F5F5F5;
}

/* Estilo del título &quot;Iniciar Sesión&quot; */
QLabel#Loginlabel {
    background-color: #85BB65;
    color: white;
    font-size: 20px;
    font-weight: bold;
    border-radius: 8px;
    padding: 8px;
}

/* Estilo de los labels */
QLabel {
    font-size: 14px;
    color: #333333;
}

/* Estilo de los campos de entrada */
QLineEdit {
    border: 2px solid #85BB65;
    border-radius: 5px;
    padding: 5px;
    font-size: 14px;
}
QLineEdit:focus {
    border: 2px solid #6FA24C;
}

/* Estilo de los botones */
QPushButton {
    background-color: #85BB65;
    color: white;
    font-size: 16px;
    font-weight: bold;
    border-radius: 5px;
    padding: 8px;
}
QPushButton:hover {
    background-color: #6FA24C;
}
QPushButton:pressed {
    background-color: #5C8E3E;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnSuscripciones">
       <property name="text">
        <string>Suscripciones</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnHistorialPagos">
       <property name="text">
        <string>Historial de Pagos</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMetodosPago">
       <property name="text">
        <string>Métodos de Pago</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnNotificaciones">
       <property name="text">
        <string>Notificaciones</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="QWidget" name="pageSuscripciones">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="listSuscripciones"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnAgregarSuscripcion">
         <property name="text">
          <string>Agregar Suscripción</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEliminarSuscripcion">
         <property name="text">
          <string>Eliminar Suscripción</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnModificarSuscripcion">
         <property name="text">
          <string>Modificar Suscripción</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageHistorialPagos"/>
     <widget class="QWidget" name="pageMetodosPago">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QListWidget" name="listMetodosPago"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnAgregarMetodoPago">
         <property name="text">
          <string>Agregar Método de Pago</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEliminarMetodoPago">
         <property name="text">
          <string>Eliminar Método de Pago</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageNotificaciones"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
